phpcs.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0"?>
 <ruleset name="code-sniffer">
     <config name="installed_paths" value="../../spryker/code-sniffer"/>
+    <config name="php_version" value="70300"/>
```

Comment:
I wasn't aware that this is possible - thank you, @szepeviktor!

Reply:
You're welcome.